src/styles/reset.scss

```diff
@@ -0,0 +1,5 @@
+* {
```

Comment:
It's not big deal, but it would be better to use margin and padding reset for the specific tags.